patches/tModLoader/Terraria/Social/Steam/WorkshopHelper.cs.patch

```diff
@@ -186,7 +186,8 @@
 +					return;
 +
 +				do {
-+					SteamedWraps.ForceCallbacks();
++					// @TODO: This is blocking
```

Comment:
@Solxanich to review with @DarioDaF 
All of these are single-thread calls on a dedicated server.